roles/sro_deploy_custom_image/tasks/deploy.yml

```diff
@@ -0,0 +1,26 @@
+- block:
+  - name: Git checkout
+    git:
+      repo: "{{ sro_git_repo }}"
+      dest: /tmp/special-resource-operator
+      version: "{{ sro_git_ref }}"
+
+  - name: Deploy the custom version of the special resource operator
+    shell:
+      make deploy IMAGE="{{ sro_custom_image }}" NAMESPACE="openshift-special-resource-operator"
+          > {{ artifact_extra_logs_dir }}/sro_deploy_custom.log 2>&1
+    args:
+      chdir: /tmp/special-resource-operator
+
+  rescue:
+  - name: Get the state of the special resource operator pod (debug)
+    command: oc get pods -n openshift-special-resource-operator
+    failed_when: false
+
+  - name: Get the logs of SRO the pod (debug)
+    command:
+       oc logs -n openshift-special-resource-operator deployment.apps/special-resource-controller-manager -c manager
```

Comment:
can you write these logs to a file, eg `{{ artifact_extra_logs_dir }}/sro_controller_manager.log`?

overall, I try to keep the task output concise, and store everything that can be useful for troubleshooting and/or understanding what happened during the execution inside files in `{{ artifact_extra_logs_dir }}`

Reply:
This file is different now, but there is extra logs dir in some key places.